src/components/common/ListPeopleAndRoles.vue

```diff
@@ -264,6 +265,17 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   // Local properties
   protected activeIndex: number // is NaN for new org/person
 
+  /**
+  * PersonList is filtered by RoleType.
+  */
```

Comment:
Lines 269 and 270 should be indented by 1 character (to line up with the first star on line 268).

Reply:
Will. Fix. Somehow `npm run lint` did not complain about this